api/base/serializers.py

```diff
@@ -266,21 +304,15 @@ def resolve_url(self, obj):
         return waterbutler_api_url_for(obj.node._id, obj.provider, obj.path, **self.kwargs)
 
 
-class NodeFileHyperLink(JSONAPIHyperlinkedIdentityField):
-    def __init__(self, kind=None, kwargs=None, **kws):
+class NodeFileHyperLink(RelationshipField):
```

Comment:
It was this way before but I feel like this should end in Field just to follow DRF style.